src/index.scss

```diff
@@ -0,0 +1,125 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+$grey-color: #616070;
+$blue-color: #00acdc;
+$white-color: #fff;
+$dark-color: #060b35;
+
+:root {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $white-color;
+  margin: 0;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-color;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+    margin: 4px 84px 0 16px;
+  }
+
+  .stars {
+    margin: 16px 16px 0;
+    display: flex;
+    justify-content: space-between;
+
+    &__list {
+      display: flex;
+    }
+
+    &__list-item {
+      background-image: url(./images/star.svg);
+      background-position: 50%;
+      background-repeat: no-repeat;
+      width: 16px;
+      height: 16px;
+      list-style: none;
+      margin-right: 4px;
+    }
+
+    &__list :nth-child(-n+4) {
+      background-image: url(./images/star-active.svg);
+    }
+
+    &__reviews {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      align-self: center;
+      color: $dark-color;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0 16px;
```

Comment:
its better to make 16px margings for the whole card, so you shouldn't set it for each element inside it